docs/framework/docker/aspnetmvc.md

```diff
@@ -227,15 +219,7 @@ To remove the container, issue a `docker rm` command:
 docker rm randomanswers
 ```
 
-## Summary
-
-In this topic, you've seen the steps to move and run an existing
-ASP.NET MVC application in a Windows Server container. Running an
-existing application does not require any changes to your
-application. You need to run the tasks to publish your application,
-build a Docker image, and start that image in a new container. Leveraging
-Windows Server Containers are the easiest path to migrate your existing
-applications to this environment.
+## Resources
```

Comment:
Although, I only use those in cases where the same link is repeated several times in the content, which doesn't seem to be the case here.